<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1520409773672" stop="1520409864369" version="1.4.8">
    <name>Default suite</name>
    <title>Default suite : Default test[]</title>
    <test-cases>
        <test-case start="1520409796754" stop="1520409833519" status="passed">
            <name>validateLogin</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="CHDSEZ97152L.local"/>
                <label name="thread" value="25135@CHDSEZ97152L.local.main"/>
            </labels>
        </test-case>
        <test-case start="1520409796754" stop="1520409833519" status="passed">
            <name>validateLogin</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="CHDSEZ97152L.local"/>
                <label name="thread" value="25135@CHDSEZ97152L.local.main"/>
            </labels>
        </test-case>
        <test-case start="1520409833606" stop="1520409855781" status="passed">
            <name>validateSearchItem</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="CHDSEZ97152L.local"/>
                <label name="thread" value="25135@CHDSEZ97152L.local.main"/>
            </labels>
        </test-case>
        <test-case start="1520409833606" stop="1520409855781" status="passed">
            <name>validateSearchItem</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="CHDSEZ97152L.local"/>
                <label name="thread" value="25135@CHDSEZ97152L.local.main"/>
            </labels>
        </test-case>
        <test-case start="1520409855789" stop="1520409862873" status="passed">
            <name>validateItemBought</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="CHDSEZ97152L.local"/>
                <label name="thread" value="25135@CHDSEZ97152L.local.main"/>
            </labels>
        </test-case>
        <test-case start="1520409855789" stop="1520409862873" status="passed">
            <name>validateItemBought</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="CHDSEZ97152L.local"/>
                <label name="thread" value="25135@CHDSEZ97152L.local.main"/>
            </labels>
        </test-case>
        <test-case start="1520409862879" stop="1520409864203" status="broken">
            <name>setAddressDetails</name>
            <failure>
                <message>NoSuchElementException: Can't locate an element by this strategy: Locator map: 
- native content: "By.id: address1" 
- html content: "by id or name "aeAddressLine1""
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 17:00:58'
System info: host: 'CHDSEZ97152L.local', ip: '10.64.105.118', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.10.5', java.version: '1.8.0_45'
Driver info: driver.version: unknown</message>
                <stack-trace>org.openqa.selenium.NoSuchElementException: Can't locate an element by this strategy: Locator map: 
- native content: "By.id: address1" 
- html content: "by id or name "aeAddressLine1""
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 17:00:58'
System info: host: 'CHDSEZ97152L.local', ip: '10.64.105.118', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.10.5', java.version: '1.8.0_45'
Driver info: driver.version: unknown
	at io.appium.java_client.pagefactory.AppiumElementLocator.findElement(AppiumElementLocator.java:169)
	at io.appium.java_client.pagefactory.ElementInterceptor.intercept(ElementInterceptor.java:30)
	at io.appium.java_client.android.AndroidElement$$EnhancerByCGLIB$$3186ea0.sendKeys(&lt;generated&gt;)
	at com.infy.pages.AddressDetails.setAddressLine1(AddressDetails.java:45)
	at com.infy.actions.ActionControllerAddress.enterAddress(ActionControllerAddress.java:18)
	at com.infy.login.EbayTestCases.setAddressDetails(EbayTestCases.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="CHDSEZ97152L.local"/>
                <label name="thread" value="25135@CHDSEZ97152L.local.main"/>
            </labels>
        </test-case>
        <test-case start="1520409862879" stop="1520409864203" status="broken">
            <name>setAddressDetails</name>
            <failure>
                <message>NoSuchElementException: Can't locate an element by this strategy: Locator map: 
- native content: "By.id: address1" 
- html content: "by id or name "aeAddressLine1""
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 17:00:58'
System info: host: 'CHDSEZ97152L.local', ip: '10.64.105.118', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.10.5', java.version: '1.8.0_45'
Driver info: driver.version: unknown</message>
                <stack-trace>org.openqa.selenium.NoSuchElementException: Can't locate an element by this strategy: Locator map: 
- native content: "By.id: address1" 
- html content: "by id or name "aeAddressLine1""
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 17:00:58'
System info: host: 'CHDSEZ97152L.local', ip: '10.64.105.118', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.10.5', java.version: '1.8.0_45'
Driver info: driver.version: unknown
	at io.appium.java_client.pagefactory.AppiumElementLocator.findElement(AppiumElementLocator.java:169)
	at io.appium.java_client.pagefactory.ElementInterceptor.intercept(ElementInterceptor.java:30)
	at io.appium.java_client.android.AndroidElement$$EnhancerByCGLIB$$3186ea0.sendKeys(&lt;generated&gt;)
	at com.infy.pages.AddressDetails.setAddressLine1(AddressDetails.java:45)
	at com.infy.actions.ActionControllerAddress.enterAddress(ActionControllerAddress.java:18)
	at com.infy.login.EbayTestCases.setAddressDetails(EbayTestCases.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="CHDSEZ97152L.local"/>
                <label name="thread" value="25135@CHDSEZ97152L.local.main"/>
            </labels>
        </test-case>
        <test-case start="1520409864212" stop="1520409864213" status="canceled">
            <name>getResult[[TestResult name=setAddressDetails status=FAILURE method=EbayTestCases.setAddressDetails()[pri:3, instance:com.infy.login.EbayTestCases@d44fc21] output={null}]]</name>
            <description></description>
            <failure>
                <message>NullPointerException: null</message>
                <stack-trace>java.lang.NullPointerException
	at com.infy.login.ExtentReportsClassVersion3.getScreenhot(ExtentReportsClassVersion3.java:72)
	at com.infy.login.ExtentReportsClassVersion3.getResult(ExtentReportsClassVersion3.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:703)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="CHDSEZ97152L.local"/>
                <label name="thread" value="25135@CHDSEZ97152L.local.main"/>
            </labels>
        </test-case>
        <test-case start="1520409864219" stop="1520409864219" status="canceled">
            <name>setPaymentDetails</name>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="CHDSEZ97152L.local"/>
                <label name="thread" value="25135@CHDSEZ97152L.local.main"/>
            </labels>
        </test-case>
        <test-case start="1520409864219" stop="1520409864219" status="canceled">
            <name>setPaymentDetails</name>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:148)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="CHDSEZ97152L.local"/>
                <label name="thread" value="25135@CHDSEZ97152L.local.main"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
